app/models/concerns/line_item_stock_changes.rb

```diff
@@ -1,19 +0,0 @@
-# frozen_string_literal: true
-
-# Rails 5 introduced some breaking changes to these built-in methods, and the new versions
-# no longer work correctly in relation to decrementing stock with LineItems / VariantOverrides.
-# The following methods re-instate the pre-Rails-5 versions, which work as expected.
-# https://apidock.com/rails/v4.2.9/ActiveRecord/Persistence/increment%21
-# https://apidock.com/rails/v4.2.9/ActiveRecord/Persistence/decrement%21
-
-module LineItemStockChanges
-  extend ActiveSupport::Concern
-
-  def increment!(attribute, by = 1)
-    increment(attribute, by).update_attribute(attribute, self[attribute])
-  end
-
-  def decrement!(attribute, by = 1)
-    decrement(attribute, by).update_attribute(attribute, self[attribute])
-  end
-end
```

Comment:
That's confusing. Matt would have had a good reason to write this.

* https://github.com/openfoodfoundation/openfoodnetwork/commit/219c982db7ee6c7c8cc6b20a376589a9f09aa433 of https://github.com/openfoodfoundation/openfoodnetwork/pull/6913

Since it was a backport, we don't exactly know to which future change it was referring to. And maybe that other branch changed along the way. I can't find any use of this either.

Line item quantity changes are not as critical as stock level changes. They need to be synchronised with stock changes but there's no normal case in which a line items would get updated by two different processes.

Reply:
So, should I revert this part too with some comment like with `variant_override.rb` below ?